cluster/common/flux/deploy_flux.sh

```diff
@@ -58,8 +99,23 @@ fi
 #   release name: flux
 #   git url: where flux monitors for manifests
 #   git ssh secret: kubernetes secret object for flux to read/write access to manifests repo
+HELM_BIN="$TMP_DIR/linux-amd64/helm"
 echo "generating flux manifests with helm template"
-if ! helm template . --name "$RELEASE_NAME" --namespace "$KUBE_NAMESPACE" --values values.yaml --set image.repository="$FLUX_IMAGE_REPOSITORY" --set image.tag="$FLUX_IMAGE_TAG" --output-dir "./$FLUX_MANIFESTS" --set git.url="$GITOPS_SSH_URL" --set git.branch="$GITOPS_URL_BRANCH" --set git.secretName="$KUBE_SECRET_NAME" --set git.path="$GITOPS_PATH" --set git.pollInterval="$GITOPS_POLL_INTERVAL" --set git.label="$GITOPS_LABEL" --set registry.acr.enabled="$ACR_ENABLED" --set syncGarbageCollection.enabled="$GC_ENABLED"; then
+if ! $HELM_BIN template $RELEASE_NAME . \
+        --values values.yaml \
+	--namespace "$KUBE_NAMESPACE" \
```

Comment:
Looks like indentation is off here.

Reply:
Rather than being a single line which was hard to read, I expanded it to multiple lines.  But there is a spacing issue that I see, yes.